fe4u-boilerplate-main/src/html/Task1.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="description" content="Лабораторная робота №1 с 4 подпунктами">
+    <meta name="msapplication-tap-highlight" content="no">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Project</title>
+
+</head>
+<body>
+
+<h1>Task 1</h1>
+<hr>
+
+<h2> Form for add teacher</h2>
+<div>Створити html файл з формою для додавання викладача. Форма
+    повинна містити наступні поля: Повне ім’я, стать, дата народження, email,
+    телефон, бажаний колір фону, країна, місто, коментар.
+</div>
+<hr>
+<form name="add_teacher"></form>
+
+<label for="fname">First name:</label><br>
+<input type="text" id="fname" name="fname"><br>
```

Comment:
Please delete all `<br>` tags.
If you want an empty line after the `label`, do it with CSS: 
```
input[type=text] {
  display: block;
}
```
The `input`s are usually inside `label`s:
```
<label>First name:
    <input type="text" id="fname" name="fname">
</label>
```
Making `input`s outside `label`s is not a mistake, but putting it inside is more common approach.

Reply:
By the way, you didn't imported `css` files to `html`, so they don't have any sense in the project.